.test-infra/jenkins/job_LoadTests_ParDo_Python_37.groovy

```diff
@@ -0,0 +1,71 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import CommonJobProperties as commonJobProperties
+import LoadTestsBuilder as loadTestsBuilder
+import PhraseTriggeringPostCommitBuilder
+
+def now = new Date().format("MMddHHmmss", TimeZone.getTimeZone('UTC'))
+
+def loadTestConfigurations = { datasetName -> [
```

Comment:
At some point, we discussed a common place for all load tests configuration. Such map then could be passed to different files with tests and filtered according to needs. Do you think having such a centralized config is a good idea? 

We do not have to do that now - let's create a separate Jira ticket and refactor globally for all tests if that makes sense.

Reply:
It's definitely worth considering. It's bad that this configuration is repeated each time we need a slightly different version of the test. I'll create a jira ticket for the future refactor.

Reply:
Link: https://issues.apache.org/jira/browse/BEAM-8941